src/main/java/spireMapOverhaul/zones/brokenSpace/patches/BrokenSpaceRenderPatch.java

```diff
@@ -0,0 +1,196 @@
+            if (shouldRenderBrokenSpaceShader(__instance)) {
+                float strength = 1.0F;
+
+                if (__instance.hb.hovered) {
+                    strength = 0.2F;
+                }
+                StopFbo(sb, strength, RandomTimeField.randomTime.get(__instance));
+
+            }
+        }
+    }
+
+
+
+    public static void StartFbo(SpriteBatch sb) {
+        sb.end();
+
+        fbo.begin();
+        Gdx.gl.glClearColor(0.0F, 0.0F, 0.0F, 0.0F);
+        Gdx.gl.glClear(GL20.GL_COLOR_BUFFER_BIT | GL20.GL_DEPTH_BUFFER_BIT);
+        sb.setBlendFunction(GL20.GL_SRC_ALPHA, GL20.GL_ONE_MINUS_SRC_ALPHA);
+        sb.begin();
+    }
+
+    public static void StopFbo(SpriteBatch sb) {
+        StopFbo(sb, 1.0F);
+    }
+
+    public static void StopFbo(SpriteBatch sb, float strength) {
+        StopFbo(sb, strength, 0.0F, 0.0F);
+    }
+
+    private static void StopFbo(SpriteBatch sb, float strength, float timerOffset) {
+        StopFbo(sb, strength, timerOffset, 0.03F);
+    }
+
+
+    public static void StopFbo(SpriteBatch sb, float strength, float timerOffset, float chrAb) {
+        sb.flush();
+        fbo.end();
+
+        TextureRegion region = new TextureRegion(fbo.getColorBufferTexture());
+        region.flip(false, true);
+
+
+        sb.setShader(brokenSpaceShader);
+        sb.setColor(Color.WHITE);
+        brokenSpaceShader.setUniformf("u_time", shaderTimer + timerOffset);
+        brokenSpaceShader.setUniformf("u_strength", strength);
+        brokenSpaceShader.setUniformf("u_chrAb", chrAb);
+
+        sb.draw(region, 0, 0);
+        sb.setShader(null);
+    }
+
+
+    private static boolean shouldRenderBrokenSpaceShader(RewardItem __instance) {
+        return inBrokenSpace() && (__instance.type == RewardItem.RewardType.CARD);
+
+    }
+
+    private static boolean shouldRenderBrokenSpaceShader(AbstractCard __instance) {
+        return inBrokenSpace() && isUnnatural(__instance) && !adp().masterDeck.contains(__instance);
+    }
+
+    private static boolean shouldRenderBrokenSpaceShader(AbstractRelic __instance, boolean isShop) {
+        return inBrokenSpace() && isUnnatural(__instance, isShop) && !adp().relics.contains(__instance);
+    }
+
+    private static boolean isUnnatural(AbstractRelic __instance, boolean isShop) {
+        boolean isUnnatural = __instance.tier == AbstractRelic.RelicTier.SPECIAL || __instance.tier == AbstractRelic.RelicTier.BOSS;
+        if (!isShop) {
+            isUnnatural = isUnnatural || __instance.tier == AbstractRelic.RelicTier.SHOP;
+
+        }
+
+        return isUnnatural;
+    }
+
+
+    private static boolean isUnnatural(AbstractCard __instance) {
+        return __instance.color != adp().getCardColor() && BrokenSpaceZone.UnnaturalCardField.unnatural.get(__instance);
+    }
+
+
+    private static boolean inBrokenSpace() {
+        return ZonePatches.currentZone() != null && ZonePatches.currentZone().id.equals("BrokenSpace");
+    }
+
+
+    @SpirePatch(
```

Comment:
There is a scubscriber for the post render and this might be needed for other shaders as well, so could play please just add this to the main mod class instead? (SpireAnniversary6Mod)

Packmaster does the same thing